var/spack/repos/builtin/packages/neovim/package.py

```diff
@@ -136,6 +137,11 @@ class Neovim(CMakePackage):
     with when("@0.8:"):
         depends_on("libvterm@0.3:", type="link")
```

Comment:
should this be 0.2: then?

Reply:
It's a good point. I did the changes in two different times following two different paths.

I realised that `neovim@0.7` was not building anymore, as I stated in the comment of the conflicts.

But about the requirement for `neovim@0.8` I checked CMake...and from the commit history it seems they switched the requirement from `libvterm@0.1` to `libvterm@0.3` , skipping `libvterm@0.2` (see [master](https://github.com/neovim/neovim/blame/master/CMakeLists.txt#L525) and [previous change](https://github.com/neovim/neovim/blame/e6c214033a4fadf60faf99e95f8e9787e3c5e630/CMakeLists.txt#L505))

I didn't realise that until your comment 😅

Reply:
Sounds reasonable.  I'll approve.